react/features/connection-indicator/components/ConnectionIndicator.js

```diff
@@ -68,20 +67,21 @@ class ConnectionIndicator extends Component {
         connectionStatus: React.PropTypes.string,
 
         /**
-         * Whether or not the displays stats are for local video.
+         * Whether or not clicking the indicator should display a popover for
+         * more details.
          */
-        isLocalVideo: React.PropTypes.bool,
+        enableStatsDisplay: React.PropTypes.bool,
 
         /**
-         * The callback to invoke when the hover state over the popover changes.
+         * Whether or not the displays stats are for local video.
          */
-        onHover: React.PropTypes.func,
+        isLocalVideo: React.PropTypes.bool,
 
         /**
-         * Whether or not the popover should display a link that can toggle
-         * a more detailed view of the stats.
+         * Relative to the icon from where the popover for more connection
+         * details should display.
          */
-        showMoreLink: React.PropTypes.bool,
+        statsPopoverPosition: React.PropTypes.string,
```

Comment:
enableStatsDisplay is to prevent clicks from causing the stats popover to display in filmstrip only mode. The statsPopoverPosition is necessary for nudging the stats display for local video in vertical filmstrip mode.

The better design here is for ConnectionIndicator to take in children to pass into the InlineDialog for display, so that a parent can better control stats display instead of relying on ConnectionIndicator. However, this design follows existing, pre-react logic and was just easier to implement at this current moment. Future reactification work (that I will do) will revisit this.